classes/controllers/grid/users/reviewer/PKPReviewerGridHandler.inc.php

```diff
@@ -1093,6 +1140,41 @@ protected function _getAuthorDeniedAnonymousOps()
             'gossip',
         ];
     }
+
+    /**
+     * Creates and sends email to the reviewer
+     * @param Mailable $mailable
+     * @param string $emailBody
+     * @param EmailTemplate $template
+     * @param User $sender
+     * @param User $reviewer
+     */
+    protected function createMail(Mailable $mailable, string $emailBody, EmailTemplate $template, User $sender, User $reviewer) : void
+    {
+        $templateData = array_merge($mailable->viewData, ['reviewerName' => $mailable->viewData['userFullName']]);
+        $mailer = PKPContainer::getInstance()['mailer'];
```

Comment:
I think you can use `App::make('mailer');` here or maybe `app('mailer')` to get the mailer.

Reply:
Rewrote the function, compilation is done inside mailable class